### DISCUSSION TITLE
Documentation says WHAT to configure, but not How/Where to configure

### DISCUSSION THREAD

**Discussion by Dweinbach** [tsightler/ring-mqtt#879] [Support Q&A]
Hi:

Relatively new to Home Assistant. I have ring-MQTT installed and running, but I want to enable camera support. I've clicked through the docs numerous times and I can't figure out WHERE to set enable_cameras. Maybe add something at the top of the "Configuration Details" wiki page for noob's like me? in the meantime, help a brotha out?

**Comment by jeffgraham-hub** [comment]
The option is in the configuration section of the ring-mqtt addon.
![Screenshot_20240727_102111_Home Assistant](https://github.com/user-attachments/assets/1f167ebf-cd12-4c36-9481-a0fafd0b2d81)

**Comment by tsightler** [comment]
> Hi:
> 
> Relatively new to Home Assistant. I have ring-MQTT installed and running, but I want to enable camera support. I've clicked through the docs numerous times and I can't figure out WHERE to set enable_cameras. Maybe add something at the top of the "Configuration Details" wiki page for noob's like me? in the meantime, help a brotha out?

This project documents ring-mqtt, not Home Assistant.  The ring-mqtt addon configuration options are set in exactly the same way as every other addon, on the addons configuration tab.
